plugins/modules/credential.py

```diff
@@ -201,6 +221,19 @@ def main() -> None:
         else (controller.get_item_name(credential) if credential else name)
     )
 
+    try:
+        result = controller.copy_if_needed(
+            name,
+            copy_from,
+            endpoint=f"eda-credentials/{credential['id']}/copy",
+            item_type="credential",
+        )
+        module.exit_json(changed=True)
+    except KeyError:
+        module.fail_json(msg="Unable to access ID of the credential to copy from: {e}")
```

Comment:
I think this could break the module if an unexisting credential is passed in the `copy_from` field, as `e` would never exist. So I guess we should either bind the exception to `e` or remove it from the fail message.

Reply:
I swear I thought I added e in the exception. But updated it now, the output looks like the following
```

TASK [Copy an existing EDA Credential] ****************************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Unable to access 'id' of the credential to copy from"}
```